src/api/app/controllers/webui/projects/pulse_controller.rb

```diff
@@ -4,26 +4,30 @@ class PulseController < WebuiController
       before_action :lockout_spiders, only: [:show]
       before_action :set_project
       before_action :set_range
-      before_action :set_pulse
 
       def show
-        @pulse = @project.project_log_entries.page(params[:page])
+        respond_to do |format|
+          format.js do
+            set_pulse
+          end
+          format.html
+        end
       end
 
       private
 
       def set_range
         @range = params[:range] == 'month' ? 'month' : 'week'
```

Comment:
`@range` is not used anymore so it can be inlined in the `case` below

Reply:
Actually it is used in https://github.com/openSUSE/open-build-service/blob/master/src/api/app/views/webui/projects/pulse/show.html.haml#L19

Reply:
right, so I'll shut up now. Nothing to see here, please keep going :laughing: